test/test_train_imagenet.py

```diff
@@ -71,10 +72,24 @@
 }
 
 
+def get_scheduler_for_model(model_name, optimizer, num_steps_per_epoch):
+  if model_name == 'resnet50':
+    return WarmupAndExponentialDecayScheduler(
+      optimizer, num_steps_per_epoch, divide_every_n_epochs=20, divisor=5)
+  else:
+    return None
+
+
 def get_model_property(key):
   return MODEL_PROPERTIES.get(FLAGS.model, MODEL_PROPERTIES['DEFAULT'])[key]
 
 
+def should_report_lr(current_device, devices):
```

Comment:
this seems like a useful util function as well, can be moved to test_utils.

Reply:
moved to test_utils and renamed